backend/api/Areas/Project/Models/Search/ProjectModel.cs

```diff
@@ -70,6 +72,21 @@ public class ProjectModel : Api.Models.BaseModel
         /// get/set - The owning subagency code.
         /// </summary>
         public string SubAgencyCode { get; set; }
+
+        /// <summary>
+        /// get/set - Project properties
+        /// </summary>
+        public IEnumerable<ProjectPropertyModel> Properties { get; set; } = new List<ProjectPropertyModel>();
+
+        /// <summary>
+        /// get/set - User Id
+        /// </summary>
+        public string UpdatedBy { get; set; }
```

Comment:
/nit is is the User id or the display name?

Reply:
DisplayName